### PR TITLE
[API] Build and Run commands for CapsulaHub CLI

### PR DESCRIPTION
**Pull request by dmitriyslaym**

### CODE REVIEW

File: src/api/CapsulaHubArgs.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/**
+ * The arguments that are used for "run" and "build" command of the app
+ */
+import { ConfigurationType } from './ConfigurationType';
+
+export interface CapsulaHubArgs {
+  /**
+   * The path to the configuration (the format of the path depends on configurationType)
+   */
+  configurationPath: string;
```
Comment (stephanebenayoun) [inline src/api/CapsulaHubArgs.ts:10]: Not clear what is this argument `configurationPath`.

File: src/api/CapsulaHubArgs.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  /**
+   * The type of configuration provider
+   * @default "httpFile"
+   */
+  configurationType?: ConfigurationType;
+  /**
+   * The token that will be used for auth
```
Comment (stephanebenayoun) [inline src/api/CapsulaHubArgs.ts:17]:
```suggestion
   * The token that will be used to create the configuration service
```

File: src/api/CapsulaHubArgs.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+   */
+  token?: string;
+  /**
+   * Relative path to the output folder
+   * @default "./dist"
+   */
+  output?: string;
```
Comment (stephanebenayoun) [inline src/api/CapsulaHubArgs.ts:24]: is this relevant for run command ? I think only for build

**Comment by stephanebenayoun** [comment]
Please open an issue before PR

File: src/api/CapsulaHubArgs.ts (hunk 1 above)
Comment (dmitriyslaym) [reply src/api/CapsulaHubArgs.ts:10]: configurationPath === token

**Pull request closed without merging**